portafly/src/components/pages/accounts/AccountsIndexPage.tsx

```diff
@@ -43,13 +48,7 @@ const AccountsIndexPage: React.FunctionComponent = () => {
           </FlexItem>
           <FlexItem breakpointMods={[{ modifier: FlexModifiers['align-right'] }]}>
             <CreateAccountButton />
-            <Button
-              variant="link"
-              icon={<ExportIcon />}
-              aria-label={t('export_accounts_button_aria_label')}
-            >
-              {t('export_accounts_button')}
-            </Button>
+            {accounts && <ExportAccountsButton data={accounts} />}
```

Comment:
How about disabling instead of hiding?
@cntlsn opinion from UX perspective?

Reply:
That makes total sense, what do you think @cntlsn ?